src/main/java/mate/academy/Main.java

```diff
@@ -2,17 +2,33 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.OrderService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy.Main");
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+
+        ShoppingCart shoppingCart = new ShoppingCart();
```

Comment:
Why is empty? Let's add movieSession to shopService, get shop cart by User and completer order for this user